R/remotebmi/README.md

```diff
@@ -0,0 +1,78 @@
+
+# remotebmi
+
+<!-- badges: start -->
+<!-- badges: end -->
+
+The goal of remotebmi is to allow a model with BMI interface written in R to be called from a Python client via a http json webservice.
+
+## Installation
+
+You can install the development version of remotebmi from [GitHub](https://github.com/) with:
+
+``` r
+# install.packages("pak")
+pak::pak("github::eWaterCycle/remotebmi/R/remotebmi")
```

Comment:
This command does not work, 

```
> pak::pak("github::eWaterCycle/remotebmi/R/remotebmi")
! Using bundled GitHub PAT. Please add your own PAT using `gitcreds::gitcreds_set()`.
Error:                                                                          
! error in pak subprocess
Caused by error: 
! Could not solve package dependencies:
* github::eWaterCycle/remotebmi/R/remotebmi: ! pkgdepends resolution error for
github::eWaterCycle/remotebmi/R/remotebmi.
Caused by error: 
! Can't find R package in GitHub repo eWaterCycle/remotebmi in directory 'R/remotebmi'
```

Is that because of this:?
> rename package from R/remotebmi to /remotebmir

Reply:
Hmm, the path is according to https://pak.r-lib.org/reference/pak_package_sources.html#github-packages-github-

I was using
```
cd R/remotebmi
R
devtools::load_all()
# and/or
devtools::install_local()
```

Reply:
Ah that uses main branch, try  `pak::pak("github::eWaterCycle/remotebmi/R/remotebmi#10")`

Reply:
I git cloned the repo and installed it from a local path. That worked fine